JoomGalleryfriends/JG4-dev
# Fix issues in multiple uploader

This PR fixes issues reported in #108 and #310. Additionally, Uppy is updated to the latest release of v3.x.
And it fixes an issue introduced in PR #312.

### How to test this PR
Try to upload images using the "multiple add" method. Issues reported in the linked issues should be fixed and no new error should appear.
